docs/release-notes/riscv64-support.rst

```diff
@@ -0,0 +1,12 @@
+:Type: Platform
+:Summary: Add RISC-V 64 support
+
+:Description:
+    In storage stage, partitions cannot be completed
+    and maybe UEFI is not recognized
```

Comment:
Um, so this is not really support, but only some part of it?

Reply:
Hmm, maybe we need someone from the non-mainstream architecture are on Fedora to chime in ? CCing @rwmjones and @nullr0ute

Reply:
I'm not at all clear on what this comment in the documentation means, but I can say that RISC-V does support partitions (obviously!) and that there is at least preliminary UEFI support although it is not widely used.

Reply:
We do run some Pungi composes that. That does incl. booting riscv64 VM with EDK2 and running anaconda. I have a patch 1+ week old. Let me cross-check.

Reply:
> Um, so this is not really support, but only some part of it?

No, I mean, before that anaconda couldn't be partitioned, now it's fine (maybe I should rewrite the release note)
and UEFI I need more test.